serverless/containers/concepts.mdx

```diff
@@ -180,33 +198,21 @@ Serverless allows you to deploy your Functions (FaaS) and Containerized Applicat
 
 Serverless.com (Serverless Framework) is a tool that allows you to deploy serverless applications without having to manage Serverless Container's API call. Write and deploy a YAML configuration file, everything else is handled automatically, even the image building.
 
-## Serverless Function
+## Serverless Containers
 
-Serverless Functions are serverless, fully managed compute services that allow you to run small, stateless code snippets or functions in response to HTTP requests or events.
+Serverless Containers is a fully managed service that enables you to run containerized applications in a scalable and serverless environment.
 
-These functions automatically scale based on demand and are designed to be lightweight, event-driven, and easily deployable, eliminating the need to worry about infrastructure management. Functions is built on top of Serverless Containers, meaning you can run your functions packaged in containers and have them scale efficiently.
+It automatically handles infrastructure management, scaling, and load balancing, allowing you to focus on writing code without worrying about servers or clusters. Serverless Containers supports any language, framework, or binary that can run in a container, and it integrates seamlessly with other Scaleway services, making it ideal for deploying modern, event-driven, and API-based applications.
 
-## Serverless Job
+## Serverless Functions
 
-Serverless Jobs are similar to Serverless Containers but are better suited for running longer workloads. See [the comparaison between Serverless products](/serverless/containers/reference-content/difference-jobs-functions-containers) for more information.
-
-## Queue trigger
-
-A queue trigger is a mechanism that connects a container to a queue created with [Scaleway Queues](/serverless/messaging/concepts/#queues), and invokes the container automatically whenever a message is added to the queue.
-
-For each message that is sent to a queue, the trigger reads the message and invokes the associated container with the message as the input parameter.
-The container can then process the message and perform any required actions, such as updating a database or sending a notification.
-
-## Rolling update
+Serverless Functions are serverless, fully managed compute services that allow you to run small, stateless code snippets or functions in response to HTTP requests or events.
```

Comment:
Maybe we can use another word here to avoid `Serverless Functions are serverless`?